common/src/main/java/net/whimxiqal/journey/data/migration/MysqlMigrationHandler.java

```diff
@@ -0,0 +1,40 @@
+package net.whimxiqal.journey.data.migration;
+
+import net.whimxiqal.journey.Journey;
+import net.whimxiqal.journey.data.DataVersion;
+import net.whimxiqal.journey.data.sql.SqlConnectionController;
+import net.whimxiqal.journey.data.sql.mysql.MySqlConnectionController;
+
+public class MysqlMigrationHandler extends SqlMigrationHandler {
+    public MysqlMigrationHandler() {
+        super(new MySqlConnectionController());
+    }
+
+    @Override
+    public void runMigration(DataVersion currentDataVersion) {
+        int currentDataVersionNumber = currentDataVersion.internalVersion;
+
+        if (currentDataVersionNumber > 0) {
+            Journey.logger().info(String.format("Migrating database version from V%o to V%o",
+                    currentDataVersionNumber,
+                    DataVersion.latest().internalVersion
+            ));
+        } else Journey.logger().info("No database version found, setting up schema.");
+
+        // All migration cases should run including and after the current version, unless otherwise stopped.
+        switch (currentDataVersion) {
+            case V000: {
+                runBatch("/data/sql/schema/mysql.sql"); // No data exists, setup schema.
+                currentDataVersionNumber = DataVersion.latest().internalVersion;
+                break;
+            }
+            case V001: {
+                runBatch("/data/sql/migration/V1-V2/mysql.sql");
+                currentDataVersionNumber = 2;
+            }
+            default: { } // Can't migrate from here, nothing needs to be done.
+        }
+
+        saveDataVersion(DataVersion.fromInt(currentDataVersionNumber));
```

Comment:
So the main point of using enums was to avoid having to use integer values at all. I want to be very intentional about doing anything when observing any given version, so I don't like passing around an integer that's updated multiple times throughout a function.

Rather, you can set a local `version = currentDataVersion`, check `version != DataVersion.V000` for the first check, and the switch case can just set `version = DataVersion.latest()`. For logging, you can just use the enum since it prints a nice name already.